app/services/efile/poll_for_acknowledgments_service.rb

```diff
@@ -90,10 +90,20 @@ def self._handle_submission_status_response(response)
       status_updates = 0
 
       doc.css('StatusRecordGrp').each do |status_record_group|
-        if status_record_group.css('SubmissionStatusTxt').text == 'Acknowledgement Received from State'
-          status_updates += 1
-          submission = EfileSubmission.find_by(irs_submission_id: status_record_group.css('SubmissionId').text)
-          submission.transition_to(:ready_for_ack, raw_response: status_record_group.to_xml)
+        status_updates += 1
+        irs_submission_id = status_record_group.css("SubmissionId").text.strip
+        status = status_record_group.css('SubmissionStatusTxt').text.strip
+        raw_response = status_record_group.to_xml
+        submission = EfileSubmission.find_by(irs_submission_id: irs_submission_id)
+
+        if ["Acknowledgement Received from State", "Received", "Ready for Pickup", "Received by State"].include?(status)
```

Comment:
I got these statuses from here: https://www.irs.gov/pub/irs-utl/mef-doc-stp-ref-guide.pdf , I'm guessing this will change once we start to get different types of response but just trying to save the metadata first

<img width="493" alt="Screenshot 2023-12-21 at 10 45 30 PM" src="https://github.com/codeforamerica/vita-min/assets/49880002/f38c61fb-6387-40c7-a11a-a9c088583921">